docs/apache-airflow/installation/supported-versions.rst

```diff
@@ -28,7 +28,7 @@ Apache Airflow version life cycle:
 +---------+-----------------+---------------+-----------------+----------------+
 | 2       | Supported       | Dec 17, 2020  | Dec 2021        | TBD            |
```

Comment:
worth also updating this line. Airflow 2 is not limited support for Dec 2021
It's TBD

Reply:
Ah yeah

Reply:
Hopefully :D

Reply:
done -https://github.com/apache/airflow/pull/20592/commits/3900e1224037d9691e0fcc82b97245775fe1d2a6